contracts/amm/IWTokenVault.sol

```diff
@@ -42,4 +42,13 @@ interface IWTokenVault {
         uint256 collateralAmount,
         uint256 wTokenAmount
     ) external;
+
+    function getLockedValue(address _ammAddress, uint256 _expirationId)
+        external
+        view
+        returns (uint256);
+
+    function getRedeemableCollateral(address _ammAddress, uint256 _expirationId)
```

Comment:
What was the purpose for this function this PR was only view functions and I just want to make sure we aren't allowing some unexpected functionality

Reply:
There will be times when user is looking at the UI while the pool has expired unclaimed wTokens. Such tokens will be excluded from the locked value, yet not included in the redeemable collateral until `claimAllExpiredTokens` is called on the AMM. This method is designed to be called as view method, but it'll simulate claiming first. Compound has similar concept with `...current` and `...stored` methods.